{
	"header": {
		"line": "CVE-2022-1623 (LibTIFF master branch has an out-of-bounds read in LZWDecode in libtif ...)",
		"name": "CVE-2022-1623",
		"description": "(LibTIFF master branch has an out-of-bounds read in LZWDecode in libtif ...)"
	},
	"annotations": [
		{
			"line": "\t{DSA-5333-1}",
			"type": "xref",
			"bugs": [
				"DSA-5333-1"
			]
		},
		{
			"line": "\t- tiff 4.4.0~rc1-1",
			"type": "package",
			"package": "tiff",
			"kind": "fixed",
			"version": "4.4.0~rc1-1"
		},
		{
			"line": "\t[buster] - tiff <not-affected> (Vulnerable code introduced later, PoCs don't trigger)",
			"type": "package",
			"release": "buster",
			"package": "tiff",
			"kind": "not-affected",
			"description": "Vulnerable code introduced later, PoCs don't trigger"
		},
		{
			"line": "\tNOTE: https://gitlab.com/libtiff/libtiff/-/commit/b4e79bfa0c7d2d08f6f1e7ec38143fc8cb11394a (v4.4.0rc1)",
			"type": "NOTE",
			"description": "https://gitlab.com/libtiff/libtiff/-/commit/b4e79bfa0c7d2d08f6f1e7ec38143fc8cb11394a (v4.4.0rc1)"
		},
		{
			"line": "\tNOTE: Introduced by: https://gitlab.com/libtiff/libtiff/-/commit/3079627ea0dee150e6a208cec8381de611bb842b (v4.4.0rc1)",
			"type": "NOTE",
			"description": "Introduced by: https://gitlab.com/libtiff/libtiff/-/commit/3079627ea0dee150e6a208cec8381de611bb842b (v4.4.0rc1)"
		},
		{
			"line": "\tNOTE: https://gitlab.com/libtiff/libtiff/-/issues/410",
			"type": "NOTE",
			"description": "https://gitlab.com/libtiff/libtiff/-/issues/410"
		}
	]
}
